src/faidx/errors.rs

```diff
@@ -0,0 +1,13 @@
+use snafu::Snafu;
```

Comment:
I am super sorry, but I just migrated error handling from snafu to thiserror (as it seems to become quite standard nowadays). Can you adapt it here?